automated/linux/ethernet/test-eth3.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+source "../test_format.sh"
+
+TEST_NAME="Test ETH-3"
+
+#Worker Host IP_ADDRESS!
+IP_ATE="192.168.168.73"
+
+echo "$TXT_Section1"
+echo "$Test_Start $TEST_NAME"
+echo "Connecting with ATE IP-Address: $IP_ATE"
+echo "$TXT_Section1"
+
+echo "$TXT_Section2"
```

Comment:
I don't understand the point of the "echo .... ". Is it possible to use LAVA to recognize the output of testcases ?